crates/std_detect/src/detect/os/windows/aarch64.rs

```diff
@@ -0,0 +1,42 @@
+//! Run-time feature detection for Aarch64 on Windows.
+
+use crate::detect::{cache, Feature};
+
+/// Try to read the features from OS API.
+pub(crate) fn detect_features() -> cache::Initializer {
+    const PF_ARM_NEON_INSTRUCTIONS_AVAILABLE: u32 = 19;
```

Comment:
Could we add a comment with a link to where you got these from? Or in which file they are? That would help with maintenance.

Reply:
This API's document is https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-isprocessorfeaturepresent, but that document isn't updated for aarch64.

Reply:
Thanks, where did you got the values for `aarch64` from ? The windows actual header files? Can those be viewed online somewhere?

Reply:
no online document.  But chromium has copy of define as https://chromium.googlesource.com/crashpad/crashpad/+/doc/compat/non_win/winnt.h

Reply:
Thanks, can you add that link to the comment as well? After that, this is good to merge!

Reply:
The latest PR is already added for this.